### PR TITLE
HOTFIX | Add temporary message

### PR DESCRIPTION
**Pull request by tuovinensanttu**
Added temporary message stating that feedback processing is congested.

### CODE REVIEW

File: hkm/static/hkm/scss/_app.scss [hunk 1]
```diff
@@ -28,17 +28,17 @@ img {
...
-color: #555 !important;  
+color: #555 !important;
 }
- 
-input[type="text"]:-ms-input-placeholder {  
-color: #555 !important;  
+
+input[type="text"]:-ms-input-placeholder {
+color: #555 !important;
 }
```
Comment (tuovinensanttu) [inline hkm/static/hkm/scss/_app.scss:42]: Sorry about these, seems that my IDE did some magic here.

File: kuvaselaamo/locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -627,6 +627,10 @@ msgstr "Feedback submitted successfully"
 msgid "Feedback error"
 msgstr "Error occurred while submitting. Try again after a while"
 
+msgid "Feedback temp"
+msgstr "Unfortunately, the processing of feedback is currently very congested due to the volume of feedback and "
+       "response times are longer than normal. We apologize for the situation!"
```
Comment (Jokinen) [inline kuvaselaamo/locale/en/LC_MESSAGES/django.po:632]: Don't know if Aki provided these translations or not, but I would say that this expression doesn't sound idiomatic in English. "We apologise for the inconvenience" or something to that degree would sounds better to my ear. Perhaps check this with Jonatan ?

**Review by Jokinen — APPROVED** [review]
Checked that the string was show in feedback view and in conjunction with the feedback form.

File: kuvaselaamo/locale/en/LC_MESSAGES/django.po (hunk 2 above)
Comment (tuovinensanttu) [reply kuvaselaamo/locale/en/LC_MESSAGES/django.po:632]: Texts weren't provided. Both Aki and Jonathan approved these, all though your suggestion sounds good. I'll toss them another message.

Comment (Jokinen) [reply kuvaselaamo/locale/en/LC_MESSAGES/django.po:632]: If they have already OK'd these, then they are fine for me as well :)

Comment (tuovinensanttu) [reply kuvaselaamo/locale/en/LC_MESSAGES/django.po]: Aki approved the change so I will be making it before merging.

**Pull request merged**